App/TopTrumps/TopTrumps.Data/Helpers/SqlHelper.cs

```diff
@@ -104,5 +104,16 @@ public string GetUpdateCardSql()
                 [ImageUrl] = @{nameof(Card.ImageUrl)}
                 WHERE [Id] = @{nameof(Card.Id)}";
         }
+
+
+        public string GetAllUserssql()
+        {
+            return @"SELECT * FROM [Users]";
+        }
+
+        public string GetAllUsersSql()
+        {
+            throw new System.NotImplementedException();
+        }
```

Comment:
We seem to have two methods here with the same name but with an uppercase `Sql` at the end. Is this an accidental duplication? The second one is the convention in C# and actually matches your interface definition as well.

Reply:
Corrected the first method to the convention and to match the interface definition